src/Essentials/test/DeviceTests/Tests/Permissions_Tests.cs

```diff
@@ -74,5 +76,18 @@ public async Task Request_NotMainThread()
 				await Assert.ThrowsAsync<PermissionException>(async () => await Permissions.RequestAsync<Permissions.LocationWhenInUse>());
 			});
 		}
+
+		[Fact]
+		public async Task StorageAndroid13AlwaysGranted()
+		{
+			if (DeviceInfo.Platform == DevicePlatform.Android && OperatingSystem.IsAndroidVersionAtLeast(33))
```

Comment:
Can this test be run on android versions lower than 33 and it confirm a status returned?